code/src/components/ClearAllButton.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import swal from 'sweetalert';
+
+import { todos } from '../reducers/todos';
+import { SectionDeleteButton, StyledInput } from '../styledComponents/styled_components';
+
+export const ClearAllButton = () => {
+  const dispatch = useDispatch();
+  
+  const handleOnClick = () => {
+    swal({
+      title:'Clearing your todos!',
+      text:'Are you sure?',
+      buttons: ['No!', 'Oh yes!'], 
+      dangerMode: true,
+      icon: 'success'
+    })
+      .then((willDelete) => {
+          if (willDelete) {
+    dispatch(todos.actions.clearAll())
+          }
+      });
+  };
+
+  return (
+    <SectionDeleteButton className='clear-all-container'>
+      <StyledInput className='clear-all-button' 
+        onClick={handleOnClick}
+        type='submit'
+        value='Clear all'
+        aria-label='{ClearButton}'>
```

Comment:
I understand the aria-label is just a regular string describing the element, so maybe the curly bracket is not needed?